pkgs/development/tools/rust/cargo-workspaces/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, fetchpatch, defaultCrateOverrides, openssl, pkg-config }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-workspaces";
+  version = "0.2.34";
+
+  src = fetchFromGitHub {
+    owner = "pksunkara";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-7oJ/jjrv2XbOyLKUqp7EMFGR+czNSXCpWI8/JX5kmH8=";
+  };
+
+  cargoSha256 = "sha256-vZyodSxQd89dyaw4lr8vsbNjztXhFVgD0FcBP8KdPss=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
+
+  sourceRoot = "source/cargo-workspaces";
+  doCheck = false;
```

Comment:
Why skipping tests?

Reply:
Good question! After a bit of struggle tests are now running 😄.